inc/classes/subscriber/Tools/class-cloudflare-subscriber.php

```diff
@@ -0,0 +1,888 @@
+<?php
+namespace WP_Rocket\Subscriber\Tools;
```

Comment:
The subscriber should be located in `WP_Rocket\Subscriber\Addons\Cloudflare`

Also note that on branch-3.5, there is already a Cloudflare subscriber created, so we don't want to duplicate code and functionalities